From c25c4e446585881fc79ad05fcc67302be252e819 Mon Sep 17 00:00:00 2001
From: Bai Ping <ping.bai@nxp.com>
Date: Tue, 21 Nov 2017 12:52:10 +0800
Subject: [PATCH 2849/5242] MLK-16920 ARM64: dts: imx8mq: Remove disp and hdmi
 power domains

commit  2e8a3eb2e9e6c88893a2eea2bb6a6ce3f272cab4 from
https://source.codeaurora.org/external/imx/linux-imx.git

Remove the disp and hdmi power domain, these two power domain can
NOT used for domain power management.

Signed-off-by: Bai Ping <ping.bai@nxp.com>
Reviewed-by: Anson Huang <Anson.Huang@nxp.com>
Tested-by: Jian Li <jian.li@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi |   16 ----------------
 1 file changed, 16 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
index f1a7aae..2c9faf01 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
@@ -231,20 +231,6 @@
 			 <&clk IMX8MQ_CLK_VPU_DEC_ROOT>;
 	};
 
-	hdmi_pd: gpc_power_domain@6 {
-		compatible = "fsl,imx8mq-pm-domain";
-		#power-domain-cells = <0>;
-		domain-id = <6>;
-		domain-name = "HDMI_PD";
-	};
-
-	disp_pd: gpc_power_domain@7 {
-		compatible = "fsl,imx8mq-pm-domain";
-		#power-domain-cells = <0>;
-		domain-id = <7>;
-		domain-name = "DISP_PD";
-	};
-
 	mipi_csi1_pd: gpc_power_domain@8 {
 		compatible = "fsl,imx8mq-pm-domain";
 		#power-domain-cells = <0>;
@@ -497,7 +483,6 @@
 		disp-mode = <16>;			/* <default mode: #16>
 							 * #16: 1920x1080p@60Hz 16:9
 							 */
-		power-domains = <&disp_pd>;
 		status = "disabled";
 	};
 
@@ -1152,7 +1137,6 @@
 					 * #95: 3840x2160p@30Hz 16:9
 					 * #97: 3840x2160p@60Hz 16:9
 					 */
-		power-domains = <&disp_pd>;
 		status = "disabled";
 	};
 
-- 
1.7.9.5

